### PR TITLE
Add get started guide

### PR DESCRIPTION
**Pull request by stefanprodan**
Fix: #48

### CODE REVIEW

File: docs/get-started/index.md [hunk 1]
```diff
@@ -214,3 +222,129 @@ service/backend    ClusterIP   10.52.10.22   <none>        9898/TCP,9999/TCP   4
 service/frontend   ClusterIP   10.52.9.85    <none>        80/TCP              3m31s
 ```
 
+!!! note
+    From this moment forward, any changes made to the webapp
+    master branch will be synchronised with the cluster.
+
+If a Kubernetes manifest is removed from source, the reconclier will remove it from your cluster. If you
```
Comment (dholbach) [inline docs/get-started/index.md:229]:
```suggestion
If a Kubernetes manifest is removed from source, the reconciler will remove it from your cluster. If you
```

File: docs/get-started/index.md [hunk 2]
```diff
@@ -214,3 +222,129 @@ service/backend    ClusterIP   10.52.10.22   <none>        9898/TCP,9999/TCP   4
...
+delete a kustomization from the fleet infra repo, the reconciler will remove all Kubernetes objects that
```
Comment (dholbach) [inline docs/get-started/index.md:230]:
```suggestion
delete a kustomization from the `fleet-infra` repo, the reconciler will remove all Kubernetes objects that
```

**Comment by dholbach** [comment]
Great work! :bowing_man: 

Here's a few thoughts I had while trying this out:

- It'd be good to spell out which scopes the personal access token exactly needs. Is it all of `repo`?
- https://github.com/dholbach/fleet-infra is created fine, initial commits are linked to @tk (this should likely be a separate issue)
- Maybe explain a simple way to set up the two clusters and what the requirements are (version, name, ???)
- How can I check my frontend, e.g. add something like `kubectl port-forward service/frontend 8080:80 -n webapp` to the instructions?
- Where in the tutorial are the two clusters used?
- Maybe add links to docs with further information on data structures or `tk <command>` docs for the bits that were added, e.g. kustomization, git source, etc.?

**Comment by stefanprodan** [comment]
> It'd be good to spell out which scopes the personal access token exactly needs. Is it all of repo?

Yes, I've added a note on that.

> Maybe explain a simple way to set up the two clusters and what the requirements are (version, name, ???)

The cluster version is specified in the requirements section.

> Where in the tutorial are the two clusters used?

I've added kind commands to each env bootstrap.

**Pull request merged**